example/data-persistence-tf/provision-database.tf

```diff
@@ -0,0 +1,5 @@
+terraform {
```

Comment:
I am probably dense, but what does this file do?

Reply:
No, not at all - it does nothing as this block is in the main.tf file.   Will remove.

Reply:
65f4ec063